docs/walkthrough.md

```diff
@@ -0,0 +1,204 @@
+max_batch_size: 0
+input: [
+  {
+    name: "data_0",
+    data_type: TYPE_FP32,
+    dims: [ 1, 3, 224, 224]
+  }
+]
+output: [
+  {
+    name: "prob_1",
+    data_type: TYPE_FP32,
+    dims: [ 1, 1000 ]
+  }
+]
+```
+
+> NOTE: As of the 22.07 release, this step is optional by default.  Both Triton and Model Analyzer support fully auto-completing the configuration file for ONNX models.
+
+3. Start server and SDK containers in the background
+
+```bash
+# Start server container in the background
+docker run -d --gpus=all --network=host -v $PWD:/mnt --name triton-server nvcr.io/nvidia/tritonserver:22.07-py3 
+
+# Start SDK container in the background
+docker run -d --gpus=all --network=host -v $PWD:/mnt --name triton-sdk nvcr.io/nvidia/tritonserver:22.07-py3-sdk
+```
+
+> NOTE: These containers can be started interactively instead, but for the sake of demonstration it is more clear to start these containers in the background and `docker exec` into them as needed for the following steps.
+
+4. Serve the model with Triton
+```bash
+# Enter server container interactively
+docker exec -ti triton-server bash
+
+# Start serving your models
+tritonserver --model-repository=/mnt/models
+```
+
+> NOTE: The `-v $PWD:/mnt` is mounting your current directory on the host into the `/mnt` directory inside the container. So if you created your model repository in `$PWD/models`, you will find it inside the container at `/mnt/models`; You can change these paths as needed. See [docker volume](https://docs.docker.com/storage/volumes/) docs for more information on how this works.
+
+5. Run Perf Analyzer on the model (in a separate shell)
+```bash
+# Enter SDK container interactively
+docker exec -ti triton-sdk bash
+
+# Benchmark model being served from step 3
+perf_analyzer -m alexnet --concurrency-range 1:4
+```
+
+```
+...
+Inferences/Second vs. Client Average Batch Latency
+Concurrency: 1, throughput: 265.147 infer/sec, latency 3769 usec
+Concurrency: 2, throughput: 890.793 infer/sec, latency 2243 usec
+Concurrency: 3, throughput: 937.036 infer/sec, latency 3199 usec
+Concurrency: 4, throughput: 965.21 infer/sec, latency 4142 usec
+```
+
+6. Run Model Analyzer to find the best configurations
+```bash
+# Enter server container interactively
+docker exec -ti triton-server bash
+
+# Stop existing tritonserver process if still running
+# because model-analyzer will start its own server
+SERVER_PID=`ps | grep tritonserver | awk '{ printf $1 }'`
+kill ${SERVER_PID}
+
+# Install model analyzer
+pip install --upgrade pip 
+pip install triton-model-analyzer wkhtmltopdf
+
+# Profile the model
+# NOTE: This may take some time, in this example it took ~10 minutes
+model-analyzer profile \
+  --model-repository=/mnt/models \
+  --profile-models=alexnet \
+  --triton-launch-mode=local \
+  --output-model-repository-path=results
+
+# Summarize the profiling results
+model-analyzer analyze --analysis-models=alexnet
+```
+
+Example Model Analyzer output summary:
```

Comment:
I think just a bit more discussion about the chart you are displaying to the user. Sure we see that we are choosing the best throughput, but why not choose the default config when it has the lowest latency? Motivate the user as to why they should read the whole chart and not just the throughput.

I like that you chose the alexnet model to get some good numbers here.